grammar.js

```diff
@@ -391,11 +401,10 @@ module.exports = grammar({
       ')',
     )),
 
-    assignment_expression: $ => prec.right(PREC.ASSIGNMENT, seq(
+    assignment_expression: $ => prec.left(PREC.ASSIGNMENT, seq(
```

Comment:
Seems to work fine, but why left associative? Although you probably know the language far better than I do (I've never written a single Pony file ;) )

Reply:
With `prec.right` it was greedily taking the expressions on the next lines as part of the rhs of the assignment too, which the original compiler doesn't do. So i opted for this behavior.